packages/api/tests/middleware/expoJudgeAccessMiddleware.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expoJudgeAccessMiddleware } from '../../src/middleware/expoJudgeAccessMiddleware';
+import { createMockRequest } from '../testUtils/expressHelpers/createMockRequest';
+import { createMockResponse } from '../testUtils/expressHelpers/createMockResponse';
+
+describe('expo Judge Access middleware', () => {
+  it('validates that the judge as access to a session', async () => {
+    const mockId = '123';
+    const mockNext = jest.fn();
+    const mockEjsCount = 22;
+    const mockContextCount = 33;
+    const mockJudge = {};
+    const req = createMockRequest({ params: { id: mockId }, judge: mockJudge as any });
+    req.entityManager.count?.mockResolvedValue(mockEjsCount);
+    req.entityManager.count?.mockResolvedValue(mockContextCount);
```

Comment:
These optional checks shouldn't be needed anymore
```suggestion
    req.entityManager.count.mockResolvedValue(mockEjsCount);
    req.entityManager.count.mockResolvedValue(mockContextCount);
```

Also using `mockResolvedValue` in each of these will result in the second overriding the first...